plugins/database/src/db_plugin.cpp

```diff
@@ -3282,8 +3304,13 @@ irods::error db_unreg_replica_op(
         }
         status = cmlExecuteNoAnswerSql(
                      "delete from R_OBJT_ACCESS where object_id=? and not exists (select * from R_DATA_MAIN where data_id=?)", &icss );
+        // If this fails, what does it mean?
```

Comment:
not sure it's actionable for us... it would be the database itself...

Reply:
i think we'd just have a dangling avu ... pointed to a no-longer-existing-dataobj...

would this result/dangling get picked up and removed by `iadmin rum`?

Reply:
I think you're right. Of course, whatever it was trying to delete here from `R_OBJT_ACCESS` would also potentially remain, assuming the `icss` struct was not modified. Would it be appropriate to perform a rollback on error here?

Reply:
upon initial inspection... latest commit e4941a0 seems correct/good.